crates/sui-protocol-config/src/lib.rs

```diff
@@ -1219,8 +1219,8 @@ pub struct ProtocolConfig {
     /// 0 was deprecated (and currently not supported), 1 is the default version.
     random_beacon_dkg_version: Option<u64>,
 
-    /// The maximum serialised transaction size (in bytes) accepted by consensus. That should be bigger than the
-    /// `max_tx_size_bytes` with some additional headroom.
+    /// The maximum serialised transaction size (in bytes) accepted by consensus. `consensus_max_transaction_size_bytes` should    
+    /// be bigger than the `max_tx_size_bytes` with some additional headroom.
```

Comment:
This is good but how about
```suggestion
    /// The maximum serialized transaction size (in bytes) accepted by consensus.
    /// This value should include space for additional metadata, on top of the `max_tx_size_bytes` value.
```

Reply:
I still explicitly mentioned the enum name, since the original purpose for me is using pronoun here is ambiguous